lib/classes/Swift/Transport/AbstractSmtpTransport.php

```diff
@@ -465,7 +465,14 @@ private function _lookupHostname()
             && $this->_isFqdn($_SERVER['SERVER_NAME'])) {
             $this->_domain = $_SERVER['SERVER_NAME'];
         } elseif (!empty($_SERVER['SERVER_ADDR'])) {
-            $this->_domain = sprintf('[%s]', $_SERVER['SERVER_ADDR']);
+            // Set the address literal tag (See RFC 5321, section: 4.1.3)
+            if (strpos($_SERVER['SERVER_ADDR'], ':') === False) {
```

Comment:
Can you use `false` and use a yoda-condition: `if (false === strpos($_SERVER['SERVER_ADDR'], ':')) {`